quic/s2n-quic-qns/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "s2n-quic-qns"
 # this in an unpublished internal crate so the version should not be changed
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
we dont publish this crate so no need to bump

Reply:
Now I noticed the comment 🤦‍♂️